.gitignore

```diff
@@ -83,6 +83,9 @@ account.data.txt
 .png-cache
 .DS_Store
 
+# Localization build files
+src/language/locales/*/*.js
```

Comment:
Both .json and .js files are required for LIngui to work and are kept on the same folder to be easier to find. I don't understand why only one of them would be in a different location. That is also how the translation files are organized in GoodSwap/GoodProtocolUI.
Should this still be changed?

Reply:
1. The .json file is necessary because that's the only way to interact with Crowdin and the .js file is necessary because that's file Lingui uses to render text.
3. OK. I will move both files to a new folder outside of the src/ folder.

Reply:
ok, thanks